src/cnc/flavors/aws/lambda-lite/1/deploy/main.sh.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+{% include "deploy_functions.sh.j2" %}
```

Comment:
I think we should do the same as we do in the build template here:
```
source {{ deployer.rendered_files_path }}/deploy-{{ service.name }}-functions.sh
```

Reply:
Commit: https://github.com/coherenceplatform/cnc/pull/90/commits/bcc63fab49021df257b003f403c76105bf4a6d6d